docs/en/development/continuous-integration.md

```diff
@@ -122,6 +238,27 @@ Builds ClickHouse in various configurations for use in further steps. You have t
 ## Special Build Check
 Performs static analysis and code style checks using `clang-tidy`. The report is similar to the [build check](#build-check). Fix the errors found in the build log.
 
+#### Running localy in docker:
```

Comment:
https://clickhouse.com/docs/en/development/build#building-in-docker

better to give this link here